kep/batch/README.md

```diff
@@ -0,0 +1,140 @@
+# Batch scheduler based on scheduling framework
```

Comment:
Can you move this out to a separate "KEP" PR so we can review the design separately?

An example is at https://github.com/kubernetes-sigs/scheduler-plugins/tree/master/kep/2-lightweight-coscheduling. In addition to a `README.md`, let's follow the community KEP conversion to put a `kep.yaml` to include the metadata there.

In terms of naming, IMO "batch" is a bit generic, can we make the folder as "podgroup-coscheduling", and the internet title here can be "Coscheduling based on PodGroup CRD", WDYT? (the wordings "based on scheduling framework" is unneeded as all proposals in this repo are based on scheduling framework)

Reply:
Good idea. I would separate the PR first. Thanks for kind suggestions

Reply:
PR updated!
@Huang-Wei